### DISCUSSION TITLE
How to define variable secret in pre-script or post-script (test) ?

### DISCUSSION THREAD

**Discussion by TheRedfoox** [hoppscotch/hoppscotch#4256] [Q&A]
How can I set a variable secret from a script ? pw.env.set doesn't seem to have any parameters for it and I can't find anything in the documentation.

**Comment by firmanadiyuda** [comment]
Try to manually define the variable secret via GUI first, and then you can use `pw.env.set('YOUR_SECRET_VARIABLE_NAME', 'value')`. It will automatically update the secret variable.
